### ISSUE TITLE
Can't install module w/ 'google-cloud-dataproc' as a dependency in a docker build

### ISSUE THREAD

**Issue by vcolano** [GoogleCloudPlatform/google-cloud-python#5636]
I'm getting an error when I try to to install a python module from it's source using `python setup.py install` where `setup.py` has as one of it's dependencies `'google-cloud-dataproc==0.1.0'`.  The error occurs when I try to build a docker container from a Dockerfile that attempts to install the module with the dependency on `google-cloud-dataproc`. This issue only started occurring since the recent update in June which specified that the `google-cloud` module should not be installed and instead the specific libraries (like `google-cloud-storage`) would need to be specified instead.

Stack trace:
```
Traceback (most recent call last):
  File "setup.py", line 25, in <module>
    url=("https://github.com/PrecisionHealthIntelligence/"
  File "/usr/local/lib/python2.7/site-packages/setuptools/__init__.py", line 129, in setup
    return distutils.core.setup(**attrs)
  File "/usr/local/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/local/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/local/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/usr/local/lib/python2.7/site-packages/setuptools/command/install.py", line 67, in run
    self.do_egg_install()
  File "/usr/local/lib/python2.7/site-packages/setuptools/command/install.py", line 117, in do_egg_install
    cmd.run()
  File "/usr/local/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 412, in run
    self.easy_install(spec, not self.no_deps)
  File "/usr/local/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 654, in easy_install
    return self.install_item(None, spec, tmpdir, deps, True)
  File "/usr/local/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 701, in install_item
    self.process_distribution(spec, dist, deps)
  File "/usr/local/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 746, in process_distribution
    [requirement], self.local_index, self.easy_install
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 786, in resolve
    new_requirements = dist.requires(req.extras)[::-1]
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2619, in requires
    "%s has no such extra feature %r" % (self, ext)
pkg_resources.UnknownExtra: googleapis-common-protos 1.6.0b4 has no such extra feature 'grpc'
```

**Comment by tseaver** [comment]
This is actually a bug in `google-api_core 0.1.4` which `google-cloud-dataproc` pins.

The fix was merged in PR #5075, and released in `google-api_core `1.2.1`.

So, we need a release of `google-cloud-dataproc` which pins a more modern version of `google-api_core`.

**Comment by tseaver** [comment]
Hmm, the `dataproc/setup.py` file was already updated in PR #4944 to allow a post-1.0 `api_core`.  That fix was added to the changelog as part of a `0.1.1` release (PR #4954), but somehow the release didn't land at pypi.org:  the most recent (only) release there is `0.1.0`.  I don't see a `dataproc-0.1.1` tag, either.

@crwilcox Can you please make a `dataproc-0.1.2` release (or maybe `0.2.0`)?

**Comment by tseaver** [comment]
@crwilcox The lack of a current release means nobody can use the `dataproc` API at all ATM.

**Comment by crwilcox** [comment]
Released 0.1.2. it seems Release 0.1.1 was never published to PyPI so that may have been the issue.
